web/modules/custom/joinup_core/config/install/tour.tour.joinup_homepage_anonymous.yml

```diff
@@ -0,0 +1,58 @@
+id: joinup_homepage_anonymous
+module: joinup_core
+label: Joinup homepage tour for anonymous users.
+langcode: en
+status: true
+dependencies: { }
+routes:
+  - route_name: 'homepage.content'
+tips:
+  welcome:
+    id: welcome
+    plugin: text
+    label: 'Welcome to Joinup'
+    body: 'On this page you can find all the content published on the platform such as news, documents, collections, etc.'
+    weight: 1
+    location: left
+    attributes:
+      data-class: 'block-recommended-content .listing .listing__item:first-child'
+  collections:
+    id: collections
+    plugin: text
+    body: 'Lists all Joinup collections, which are the main collaborative spaces organised around a common topic.'
+    weight: 2
+    location: left
+    attributes:
+      data-class: 'stats__wrapper__collections'
+  solutions:
+    id: solutions
+    plugin: text
+    body: 'Lists all Joinup solutions, which are the main collaborative spaces organised around a collection''s sub-topic.'
+    weight: 3
+    location: left
+    attributes:
+      data-class: 'stats__wrapper__solutions'
+  content:
+    id: content
+    plugin: text
+    body: 'Find all the latest Joinup updates concerning events, documents, news, and discussions.'
+    weight: 4
+    location: right
+    attributes:
+      data-class: 'stats__wrapper__content'
+  search:
+    id: search
+    plugin: text
+    body: 'Search through the entire Joinup platform and its supported content items.'
+    weight: 5
+    location: left
+    attributes:
+      data-class: 'search-bar__input'
+  account:
+    id: account
+    plugin: text
+    body: 'Sign in or out, and manage your user account.'
+    weight: 6
+    location: left
+    attributes:
+      data-class: 'anonymous-menu'
```

Comment:
There are 7 steps in the ticket but only 6 here. Why is it 1 missing?

Reply:
Oh the balloon for the contact link is included in ISAICP-4584. There were a few balloons that were identical between these two tickets and I committed it in there instead of here.